data/json/npcs/lumbermill_employees/TALK_lumbermill_fabricate.json

```diff
@@ -0,0 +1,134 @@
+      "no": "Sure, we offer various specialty cuts direct to our customers in batches of 10.  Anything larger than these planks is gonna take us a while to refresh, though.  A couple hours, maybe longer if we have to source more wood.  So, what would you like?"
+    },
+    "responses": [
+      { "text": "Nevermind.", "topic": "TALK_NONE" },
+      {
+        "text": "[10$]  I'll buy some sticks.",
+        "effect": [ { "u_buy_item": "stick", "cost": 1000, "count": 10 } ],
+        "topic": "TALK_LUMBERMILL_FABRICATE_END"
+      },
+      {
+        "text": "[20$]  I'll buy some long sticks.",
+        "effect": [ { "u_buy_item": "stick_long", "cost": 2000, "count": 10 } ],
+        "topic": "TALK_LUMBERMILL_FABRICATE_END"
+      },
+      {
+        "text": "[30$]  I'll buy some planks.",
+        "effect": [ { "u_buy_item": "2x4", "cost": 3000, "count": 10 } ],
+        "topic": "TALK_LUMBERMILL_FABRICATE_END"
+      },
+      {
+        "text": "[100$]  I'll buy some logs.",
+        "effect": [
+          { "u_buy_item": "log", "cost": 10000, "count": 10 },
+          { "u_add_effect": "lumbermill_waiting_for_log", "duration": "2 h" },
+          { "npc_add_effect": "currently_busy", "duration": "2 h" }
+        ],
+        "topic": "TALK_LUMBERMILL_FABRICATE_END",
+        "condition": { "not": { "u_has_effect": "lumbermill_waiting_for_log" } }
+      },
+      {
+        "text": "[120$]  I'll buy some wooden panels.",
+        "effect": [
+          { "u_buy_item": "wood_panel", "cost": 12000, "count": 10 },
+          { "u_add_effect": "lumbermill_waiting_for_wood_panel", "duration": "3 h" },
+          { "npc_add_effect": "currently_busy", "duration": "3 h" }
+        ],
+        "topic": "TALK_LUMBERMILL_FABRICATE_END",
+        "condition": { "not": { "u_has_effect": "lumbermill_waiting_for_wood_panel" } }
+      },
+      {
+        "text": "[240$]  I'll buy some large wooden sheets.",
+        "effect": [
+          { "u_buy_item": "wood_sheet", "cost": 24000, "count": 10 },
+          { "u_add_effect": "lumbermill_waiting_for_wood_sheet", "duration": "3 h" },
+          { "npc_add_effect": "currently_busy", "duration": "3 h" }
+        ],
+        "topic": "TALK_LUMBERMILL_FABRICATE_END",
+        "condition": { "not": { "u_has_effect": "lumbermill_waiting_for_wood_sheet" } }
+      },
+      {
+        "text": "[350$]  I'll buy some wooden beams.",
+        "effect": [
+          { "u_buy_item": "wood_beam", "cost": 35000, "count": 10 },
+          { "u_add_effect": "lumbermill_waiting_for_wood_beam", "duration": "4 h" },
+          { "npc_add_effect": "currently_busy", "duration": "4 h" }
+        ],
+        "topic": "TALK_LUMBERMILL_FABRICATE_END",
+        "condition": { "not": { "u_has_effect": "lumbermill_waiting_for_wood_beam" } }
+      }
+    ]
+  },
+  {
+    "id": "TALK_LUMBERMILL_FABRICATE_END",
```

Comment:
I'm not sure I understand why each of the purchase list dialogues get repeated, could you explain?

Reply:
They are repeated to save time; a player buying things in bulk might want to purchase more of a given thing or buy something else. They jump back to the buy screen instead of navigating back through the dialogue tree each time they wish to alter their selection, less hassle and frustration that way.